BeerProject/src/main/java/wawer/kamil/beerproject/service/impl/BeerServiceImpl.java

```diff
@@ -33,19 +39,24 @@
     private static final String THE_BREWERY_BASE_ON_ID_HAS_NOT_BEEN_FOUND = "The brewery base on id: {} has not been found";
     private static final String THE_BEER_BASE_ON_ID_HAS_NOT_BEEN_FOUND = "The beer base on id: {} has not been found";
 
+
+    private final Logger LOG = LoggerFactory.getLogger(BeerServiceImpl.class);
```

Comment:
niepotrzebny kolejny logger. Załatwia Ci już logowanie Slf4j. jeżeli ci jest potrzebny logger to spróbój go zintegrować z już istniejącym